### PR TITLE
Mobile styling applied to the disclaimer page

### PR DESCRIPTION
**Pull request by dinadeljanin**

### CODE REVIEW

File: client/src/components/ChartJs.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            ticks: {
+                suggestedMax: 150,
+                suggestedMin: 0
+                    }
+                }]
+            }
+        }
+};
+
+const ChartJs = () => {
```
Comment (friedger) [inline client/src/components/ChartJs.js]: unrelated change here, could you please remove this change.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (friedger) [inline yarn.lock]: .lock should be added to .gitignore

File: client/src/components/App.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import { connect } from 'react-redux';
 import setLoginLoading from '../redux/actions/actions'
 
+import Disclaimer from './Disclaimer';
```
Comment (friedger) [inline client/src/components/App.js]: unnecessary change here, isn't it? Try to make the PR as small as needed.

**Comment by SomeMoosery** [comment]
@dinadeljanin - how are things coming along with this? Just need a screenshot, unnecessary change removals, and a rebase off master and this should be good to merge

**Comment by SomeMoosery** [comment]
Closing this PR - styling was added by #185. We can open a new PR later if additional styling is needed
